packages/rocketchat-livechat/.app/package.json

```diff
@@ -25,7 +25,10 @@
     "bcrypt": "^2.0.1",
     "core-js": "^2.5.7",
     "jquery": "^3.3.1",
+    "meteor-node-stubs": "^0.3.3",
+    "mime-type": "^3.0.5",
     "moment": "^2.22.2",
+    "request": "^2.85.0",
```

Comment:
this seems to not be used. if so, please remove.

Reply:
Fixed.